.azure/pipelines/ci.yml

```diff
@@ -655,4 +674,4 @@ stages:
       # See https://github.com/dotnet/arcade/issues/2871
       enableSymbolValidation: false
       enableSigningValidation: false
-      publishInstallersAndChecksums: true
+      publishInstallersAndChecksums: true
```

Comment:
Don't change the final newline if you can help it. Most prefer they are present and leaving the files alone prevents thrashing back and forth.